docs/PROVISIONING_LIBVIRT.adoc

```diff
@@ -0,0 +1,186 @@
+* adapt the Libvirt specific parameters to make them compatible with your setup (especially the network)
+- export the variable `LIBVIRT_INV_VM_FILTER` to fit the libvirt names defined for your cluster's VMs, e.g. `export LIBVIRT_INV_VM_FILTER=^ocp_`.
+- if your network isn't persistent create it (see above).
+- make sure that `/tmp/authorized_keys` exists. FIXME: not sure yet for which use cases it is required, I just copy for now my own authorized keys.
+- call `ansible-playbook -i ../../inventory/sample.libvirt.example.com.d/inventory libvirt/provision.yml`.
++
+IMPORTANT: virt-install is only running synchronously because a virt-viewer UI is popping up. Close each virt-viewer once the corresponding installation has happened and not too long after.
++
+- identify the IP address of the infrastructure VM on which the route will run and start accordingly the separate dnsmasq responsible to do the wildcard DNS resolution (see above).
+- login into one of the new VMs and validate that DNS is working correctly:
+* `dig master.local` gives the correct IP address (same for all VMs)
+* `dig -x <master-ip>` works as well
+* `dig -x xxx.apps.local` gives the IP of the route/infranode.
+
+NOTE: up till now, I've worked as root to avoid complications. From here on, I'm working again as normal user on the control host.
+
+=== pre-install
+
+IMPORTANT: you need to have ssh-ed once to each node to make sure that their SSH-signature is already in your known_hosts file.
+
+Things to consider:
+
+- make sure the above preparations are still active (network, DNS, environment variables)
+- define the environment variables `RHSM_USER` and `RHSM_PASSWD` or use an activation key (TODO describe activation key / Satellite 6 approach).
++
+CAUTION: because there is no trace of OpenShift on the system, it is relatively sure that auto-attach will fail. Hence make sure `rhsm_pool` or `rhsm_pool_ids` are defined in the inventory (or on the command line).
+
+Then call `ansible-playbook -i ../../inventory/sample.libvirt.example.com.d/inventory/ pre-install.yml -e rhsm_pool='^{POOL_NAME}$'`.
+
+=== install
+
+- make sure the credentials are set in the environment, either using your RHSM credentials or adding specific ones (OREG_AUTH_USER and OREG_AUTH_PASSWORD, see inventory OSEv3.yml for details).
+- call `ansible-playbook -i ../../inventory/sample.libvirt.example.com.d/inventory/ install.yml` and wait...
+- once the playbook has run successfully, you need to enter `master.local` into the `/etc/hosts` of your workstation (unless it's using the right DNS), and then you can point your browser to `https://master.local:8443` and login as admin (using the IP address directly won't work).
+
+
+TODO:: continue to adapt / complete the following lines for Libvirt
+
+Run the `end-to-end` provisioning playbook via our link:../images/casl-ansible/[??? installer container image].
+
+[source,bash]
+----
+docker run -u `id -u` \
```

Comment:
Why do we need an AWS Access key/secret? Why are we running this from Docker when all other Ansible commands have been run without Docker?